### PR TITLE
Fix jupyter install bugs

### PR DESCRIPTION
**Pull request by colombod**
Addressing #366
--path was ignored during install
default anaconda locations not included in strategy

### CODE REVIEW

File: dotnet-interactive/CommandLine/readme.md [hunk 1]
```diff
@@ -24,6 +24,19 @@ When installing `dotnet-interactive` as a Jupyter kernel, you can also configure
...
 dotnet interactive jupyter install                             | Disabled
-dotnet interactive jupyter install --http-port-range 8001-9000 | Enabled on autoselected port between 8001 and 9000                       
+dotnet interactive jupyter install --http-port-range 8001-9000 | Enabled on autoselected port between 8001 and 9000                      
+
+ 
+
+
+## Installing kernels
+
+When installing `dotnet-interactive` it is possible to provide the path for installing the kernel specs, the path msut exists.
```
Comment (jonsequitur) [inline dotnet-interactive/CommandLine/readme.md:34]:
```suggestion
When installing `dotnet-interactive` it is possible to provide the path for installing the kernel specs. The provided path must be an existing directory.
```

File: dotnet-interactive/CommandLine/readme.md [hunk 2]
```diff
@@ -24,6 +24,19 @@ When installing `dotnet-interactive` as a Jupyter kernel, you can also configure
...
+
+| Command                                                      | Destination path
+|--------------------------------------------------------------|--------------------------------
+dotnet interactive jupyter install                             | Will use the kernelspec module if available, if it is not present will attempt ot use platform specific folders for python or anaconda paths
```
Comment (jonsequitur) [inline dotnet-interactive/CommandLine/readme.md:38]:
```suggestion
dotnet interactive jupyter install                             | Use the kernelspec module if available. If not found, attempt to use well-known platform-specific folders for Python or Anaconda .
```

File: dotnet-interactive/Properties/launchSettings.json [hunk 3]
```diff
@@ -30,6 +30,10 @@
     "dotnet-interactive-http": {
       "commandName": "Project",
       "commandLineArgs": "http"
+    },
```
Comment (jonsequitur) [inline dotnet-interactive/Properties/launchSettings.json:33]: Please revert.

File: dotnet-interactive.Tests/CommandLine/CommandLineParserTests.cs [hunk 4]
```diff
@@ -104,14 +104,12 @@ public void jupyter_install_command_parses_path_option()
...
-
-            var option = result.CommandResult.OptionResult("--path");
+            var result = _parser.InvokeAsync($"jupyter install --path {_kernelSpecInstallPath}");
 
-            using var scope = new AssertionScope();
+            var installedKernels = _kernelSpecInstallPath.GetDirectories();
 
-            option.Should().NotBeNull();
-            result.FindResultFor(option.Option).GetValueOrDefault<DirectoryInfo>().FullName.Should().Be(_kernelSpecInstallPath.FullName);
+            installedKernels.Select(d => d.Name)
```
[the remark this answers is not in the document]
Comment (colombod) [reply dotnet-interactive.Tests/CommandLine/CommandLineParserTests.cs:111]: sure

File: README.md [hunk 5]
```diff
@@ -38,7 +38,7 @@ First, make sure you have the following installed:
 > dotnet tool install -g --add-source "https://dotnet.myget.org/F/dotnet-try/api/v3/index.json" Microsoft.dotnet-interactive
 ```
 
-- Register .NET Interactive as a Jupyter kernel by running the following within your Anaconda Prompt:
+- Register .NET Interactive as a Jupyter kernel by running the following within your Anaconda Prompt (Read the [install command line documentation](dotnet-interactive/CommandLine/readme.md) for more details):
```
[the remark this answers is not in the document]
Comment (colombod) [reply README.md:41]: done

File: dotnet-interactive/CommandLine/readme.md (hunk 2 above)
Comment (colombod) [reply dotnet-interactive/CommandLine/readme.md:38]: don

File: dotnet-interactive/Properties/launchSettings.json (hunk 3 above)
Comment (colombod) [reply dotnet-interactive/Properties/launchSettings.json:33]: doing it

**Pull request merged**